tests/apple/app_store_connect/testflight/test_beta_groups.py

```diff
@@ -50,3 +50,9 @@ def test_remove_build_from_beta_groups(self):
         beta_group = ResourceId('c954ae49-625e-49cc-a8eb-3dbf8aff94c9')
 
         self.api_client.beta_groups.remove_build(beta_group, build)
+
+    def test_add_if_build_does_not_exist(self):
+        build = ResourceId('00000000-0000-0000-0000-000000000000')
+        beta_group = ResourceId('c954ae49-625e-49cc-a8eb-3dbf8aff94c9')
+
+        self.api_client.beta_groups.add_build(beta_group, build)
```

Comment:
Even if those tests are not launched automatically since they target production App Store Connect, they should function properly. As it is now, this line here will result in `AppStoreConnectApiError`, which is not handled and the test fails. Why not wrap it inside `pytest.raises` block?